### PR TITLE
add l1 nodes info to local network describe

### PR DESCRIPTION
**Pull request by felipemadero**
## Why this should be merged
current output of local network deploy, describe, start, status is misleaving, showing only
the local primary network nodes, but not the local l1 nodes (local cluster)

### CODE REVIEW

File: pkg/localnet/localnet.go [hunk 1]
```diff
@@ -33,7 +33,13 @@ func GetEndpoint() (string, error) {
 }
 
 func GetClusterInfo() (*rpcpb.ClusterInfo, error) {
-	cli, err := binutils.NewGRPCClient(
+	return GetClusterInfoWithEndpoint(binutils.LocalNetworkGRPCServerEndpoint)
+}
+
+func GetClusterInfoWithEndpoint(grpcServerEndpoint string) (*rpcpb.ClusterInfo, error) {
```
Comment (arturrez) [inline pkg/localnet/localnet.go:39]: ❤️

File: pkg/node/local.go [hunk 2]
```diff
@@ -758,7 +758,7 @@ func listLocalClusters(app *application.Avalanche, clusterNamesToInclude []strin
 	return localClusters, nil
 }
 
-func DestroyCurrentIfLocalNetwork(app *application.Avalanche) error {
+func CurrentIsLocalNetwork(app *application.Avalanche) (bool, string, error) {
```
Comment (arturrez) [inline pkg/node/local.go:761]: nit: add comment what `(bool, string, error)`  means in func return as it's not clear from the name

Comment (arturrez) [reply pkg/node/local.go:761]: maybe func name is not that great, there should be a better name like `GetLocalNetworkState` which returns clusterName and state running or not

File: pkg/node/local.go [hunk 3]
```diff
@@ -776,58 +776,43 @@ func DestroyCurrentIfLocalNetwork(app *application.Avalanche) error {
...
-	if currentlyRunningRootDir == "" {
-		return nil
+	if isLocal {
+		_ = DestroyLocalNode(app, clusterName)
 	}
-	localClusters, err := listLocalClusters(app, nil)
+	return nil
+}
+
+func StopCurrentIfLocalNetwork(app *application.Avalanche) error {
```
Comment (arturrez) [inline pkg/node/local.go:809]: nit: better name can be StopLocalNetwork which returns error if it's not running or not local 

pls feel free to ignore this comment

**Review by arturrez — APPROVED** [review]
approved with some nits

File: pkg/node/local.go (hunk 2 above)
Comment (felipemadero) [reply pkg/node/local.go:761]: renamed to ConnectedToLocalNetwork.

**Pull request merged**